mlflow/protos/service.proto

```diff
@@ -1623,6 +1634,22 @@ message GetLoggedModel {
   }
 }
 
+message SetLoggedModelTags {
+  option (scalapb.message).extends = "com.databricks.rpc.RPC[$this.Response]";
+  option (scalapb.message).extends = "com.databricks.mlflow.api.MlflowTrackingMessage";
```

Comment:
@harupy Can we remove this from the new API and all existing APIs? This actually shouldn't be present in OSS